code/modules/cargo/exports/organs_robotics.dm

```diff
@@ -110,6 +111,17 @@
 	export_types = list(/obj/item/organ/liver)
 	exclude_types = list(/obj/item/organ/liver/cybernetic, /obj/item/organ/liver/cybernetic/upgraded)
 
+/datum/export/orgains/cybernetic
+	cost = 225
+	unit_name = "cybernetic organ"
+	export_types = list(/obj/item/organ/liver/cybernetic, /obj/item/organ/lungs/cybernetic, /obj/item/organ/eyes/robotic, /obj/item/organ/heart/cybernetic)
+	exclude_types = list(/obj/item/organ/lungs/cybernetic/upgraded, /obj/item/organ/liver/cybernetic/upgraded)
+
+/datum/export/orgains/upgraded
+	cost = 275
+	unit_name = "upgraded cybernetic organ"
+	export_types = list(/obj/item/organ/lungs/cybernetic/upgraded, /obj/item/organ/liver/cybernetic/upgraded)
+
 /datum/export/orgains/tail //Shhh
```

Comment:
```suggestion
/datum/export/organs/tail //Shhh
```

Will probably need to check further for more mentions of “orgains” and switch it to the correct spelling of organs.